ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -56,6 +56,11 @@ def mean(input, dim, keepdim=False, *, out=None):
     return ivy.mean(input, axis=dim, keepdims=keepdim, out=out)
 
 
+@to_ivy_arrays_and_back
+def median(input):
```

Comment:
The tests are failing for `float16` which needs to be handled by an ivy decorator for unsupported types of this function.
You will need to import at the top of the file 
`from ivy.func_wrapper import with_unsupported_dtypes`

```suggestion
@with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
def median(input):
```